books/models/book.py

```diff
@@ -0,0 +1,19 @@
+from json import loads
+
+from django.db import models
+
+
+class Book(models.Model):
```

Comment:
@xhahid43eb I didn't like the idea of creating one model to handle all requirements. Please utilise relational database properly and create a separate table/model for every entity e.g. publisher, author, country all are separate entities.

Reply:
@xhahid43eb I am not going to review serializers and views since there is a major change going to happen after you break down entities into separate models.

Reply:
> @xhahid43eb I didn't like the idea of creating one model to handle all requirements. Please utilise relational database properly and create a separate table/model for every entity e.g. publisher, author, country all are separate entities.

For making this changes our request and response data could be changed